guides/doc-Monitoring_Guide/topics/con_satellite_metrics_overview.adoc

```diff
@@ -9,6 +9,6 @@ You can collect the following metrics from Satellite:
 * Process statistics, including memory and CPU utilization;
 * Apache HTTP Server activity statistics;
 * PostgreSQL activity statistics;
-* Satellite application statistics.
+* Satellite application telemetry data.
```

Comment:
We do refer monitoring data coming out from Foreman via statsd protocil as telemetry in other parts of the document, so let's be consistent here.